chore(icons): Update and add missing pastanaga icons from https://pastanaga.io/icons ... (#3208)
plone/volto: docs/source/developer-guidelines/icons.md

---
html_meta:
  "description": "Icons used in Volto"
  "property=og:description": "Icons used in Volto"
  "property=og:title": "Icons"
  "keywords": "Volto, Plone, frontend, React, Icons, svg, pastanaga"
---

# Icons

> Volto has pre-defined set of SVG icons from Pastanaga UI icon system and are available under [Volto icons](https://github.com/plone/volto/tree/master/src/icons).

Reviewer: ```suggestion
Volto has a pre-defined set of SVG icons from the Pastanaga UI icon system.
They are available under [Volto icons](https://github.com/plone/volto/tree/master/src/icons).

The following example shows how to display one of these icons.```

```js
import addUserSVG from '@plone/volto/icons/add-user.svg';
import { Icon } from '@plone/volto/components';

<Icon name={addUserSVG} size="24px" />;
```

```{note}

> These icons are intended to be used only in "official" Pastanaga UI, so please refrain to use them on personal projects unless they are based on Pastanaga UI.

Reviewer: ```suggestion
These icons are intended to be used only in the "official" Pastanaga UI.
Please refrain from using them on personal projects unless they are based on the Pastanaga UI.
```

Author: @stevepiercy Done!

# Icons

Volto has a pre-defined set of SVG icons from the Pastanaga UI icon system.
They are available under [Volto icons](https://github.com/plone/volto/tree/master/src/icons).

> The following example shows how to display one of these icons.```

Reviewer: It looks like some trailing backticks snuck in here.

```suggestion
The following example shows how to display one of these icons.
```

Author: Done
